packages/workflow/src/utils.ts

```diff
@@ -1,7 +1,13 @@
-/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-argument */
-export const deepCopy = <T>(source: T, hash = new WeakMap(), path = ''): T => {
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return */
+
+type Serializable = any & { toJSON?: (key?: any) => string };
```

Comment:
if use `any` in either union or intersection types they will evaluate to just `any` anyways

Reply:
i was derp. fixed.